### ISSUE TITLE
Use of unresolved identifier 'undefined'

### ISSUE THREAD

**Issue by attheodo** [apollographql/apollo-ios#48]
Hey,

I have this mutation definition:

```
mutation GetPhotoUploadURL($filename: String!) {
  
  photoGetUploadUrl(input: {
                    filename: $filename
                    content_type: "image/jpg"
                    photo_type: SELFIE
                    source: CAMERA
                    count: 1
                    })
  {
    upload_url
    photo_id
  }
}
```

And the generated API file appears to have an error:

```
 public struct Data: GraphQLMappable {
    public let photoGetUploadUrl: PhotoGetUploadUrl?

    public init(reader: GraphQLResultReader) throws {
      photoGetUploadUrl = try reader.optionalValue(for: Field(responseName: "photoGetUploadUrl", arguments: ["input": ["filename": undefined, "content_type": "image/jpg", "photo_type": "SELFIE", "source": "CAMERA", "count": 1]]))
    }

[...]
```
`Use of unresolved identifier 'undefined'`

Anything wrong with the way I am declaring my mutation?

**Comment by martijnwalraven** [comment]
No, I'm afraid this is the same bug as https://github.com/apollographql/apollo-codegen/issues/47. I should have a fix out today!

**Comment by martijnwalraven** [comment]
@attheodo: This should be fixed in `apollo-codegen` 0.10.4.
